libs/experimental/tests/integration_tests/llms/test_ollama_functions.py

```diff
@@ -52,9 +52,50 @@ def test_default_ollama_functions(self) -> None:
 
         res = model.invoke("What's the weather in San Francisco?")
 
-        function_call = res.additional_kwargs.get("function_call")
-        assert function_call
-        self.assertEqual(function_call.get("name"), "get_current_weather")
+        self.assertEqual(AIMessage, type(res))
+        tool_calls = res.__dict__["tool_calls"]
```

Comment:
why not just `res.tool_calls`?

Reply:
Went the __dict__ route for backwards compatibility. But happy to exclude it if not necessary.